packages/adapter-drizzle/src/lib/mysql.ts

```diff
@@ -266,18 +254,193 @@ export function MySqlDrizzleAdapter(
+      dataType: "string"
+      columnType: "MySqlVarChar" | "MySqlText"
+      data: string
+      notNull: boolean
+    }>
+    access_token: DefaultMyqlColumn<{
+      dataType: "string"
+      columnType: "MySqlVarChar" | "MySqlText"
+      data: string
+      driverParam: string | number
+      notNull: boolean
+    }>
+    expires_at: DefaultMyqlColumn<{
+      dataType: "number"
+      columnType: "MySqlInt"
+      data: number
+      notNull: boolean
+    }>
+    token_type: DefaultMyqlColumn<{
+      dataType: "string"
+      columnType: "MySqlVarChar" | "MySqlText"
+      data: string
+      notNull: boolean
+    }>
+    scope: DefaultMyqlColumn<{
+      dataType: "string"
+      columnType: "MySqlVarChar" | "MySqlText"
+      data: string
+      notNull: boolean
+    }>
+    id_token: DefaultMyqlColumn<{
+      dataType: "string"
+      columnType: "MySqlVarChar" | "MySqlText"
+      data: string
+      notNull: boolean
+    }>
+    session_state: DefaultMyqlColumn<{
+      dataType: "string"
+      columnType: "MySqlVarChar" | "MySqlText"
+      data: string
+      notNull: boolean
+    }>
+  }
+  dialect: "mysql"
+  schema: string | undefined
+}>
+
+export type DefaultMySqlSessionsTable = MySqlTableWithColumns<{
+  name: string
+  columns: {
+    sessionToken: DefaultMyqlColumn<{
+      columnType: "MySqlVarChar" | "MySqlText"
+      data: string
+      notNull: true
+      dataType: "string"
+    }>
+    userId: DefaultMyqlColumn<{
+      columnType: "MySqlVarChar" | "MySqlText"
+      data: string
+      notNull: true
+      dataType: "string"
+    }>
+    expires: DefaultMyqlColumn<{
+      dataType: "date"
+      columnType: "MySqlTimestamp"
+      data: Date
+      notNull: true
+    }>
+  }
+  dialect: "mysql"
+  schema: string | undefined
+}>
+
+export type DefaultMySqlVerificationTokenTable = MySqlTableWithColumns<{
+  name: string
+  columns: {
+    identifier: DefaultMyqlColumn<{
+      columnType: "MySqlVarChar" | "MySqlText"
+      data: string
+      notNull: true
+      dataType: "string"
+    }>
+    token: DefaultMyqlColumn<{
+      columnType: "MySqlVarChar" | "MySqlText"
+      data: string
+      notNull: true
+      dataType: "string"
+    }>
+    expires: DefaultMyqlColumn<{
+      dataType: "date"
+      columnType: "MySqlTimestamp"
+      data: Date
+      notNull: true
+    }>
+  }
+  dialect: "mysql"
   schema: string | undefined
 }>
 
 export type DefaultMySqlSchema = {
-  usersTable: MySqlTableFn<(typeof mysqlUsersTable)["_"]["config"]>
-  accountsTable: MySqlTableFn<(typeof mysqlAccountsTable)["_"]["config"]>
-  sessionsTable: MySqlTableFn<(typeof mysqlSessionsTable)["_"]["config"]>
-  verificationTokensTable: MySqlTableFn<
-    (typeof mysqlVerificationTokensTable)["_"]["config"]
-  >
+  usersTable: DefaultMySqlUsersTable
+  accountsTable: DefaultMySqlAccountsTable
+  sessionsTable: DefaultMySqlSessionsTable
+  verificationTokensTable: DefaultMySqlVerificationTokenTable
```

Comment:
nit - but should the default columns have some guard that they adhere to these types to avoid drifting away from public interface?

```ts
export const defaultUsersTable = mysqlTable(...) satisfies DefaultMySqlUsersTable;
```

Reply:
Added it  🚀